craft_parts/dirs.py

```diff
@@ -35,6 +35,26 @@ def work_dir(self) -> Path:
         """Return the root of the work directories used for project processing."""
         return self._work_dir
 
+    @property
+    def overlay_dir(self) -> Path:
+        """Return the directory containing work subdirectories for overlays."""
+        return self._work_dir / "overlay"
```

Comment:
Do these need to be properties?

Why not keep them simple, and just, in `__init__`, do `self.overlay_dir = self._work_dir / "overlay"` ?

Reply:
`ProjectDirs` is minimally exposed in the codebase so we may want to change all its properties to attributes.

Reply:
Done.